src/jobflow_remote/jobs/jobcontroller.py

```diff
@@ -2597,7 +2687,10 @@ def delete_flow(
             return False
         job_ids = flow["jobs"]
         if delete_output:
-            self.jobstore.remove_docs({"uuid": {"$in": job_ids}})
+            jobstore = self.jobstore
+            if flow.get("jobstore"):
+                jobstore = self.optional_jobstores[flow["jobstore"]]
```

Comment:
```suggestion
            if jobstore_name := flow.get("jobstore"):
                jobstore = self.optional_jobstores[jobstore_name]
```
Use walrus operator here ?